src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -694,6 +694,17 @@
                 "nexthop_group_threshold_type": "percentage",
                 "polling_interval": "0"
             }
+        },
+        "PORT_STORM_CONTROL": {
+            "Ethernet0|broadcast": {
+                "kbps": "10000"
+            },
+            "Ethernet0|unknown-multicast": {
+                "kbps": "30000"
+            },
+            "Ethernet0|unknown-unicast": {
+                "kbps": "20000"
+            }
```

Comment:
Please write the actual UT cases as well e.g add cases in src/sonic-yang-models/tests/yang_model_tests/tests for this feature.

Reply:
Done